packages/next-auth/src/lib/actions.test.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, it, expect, vi, beforeEach, afterEach } from "vitest"
```

Comment:
I think we can move the tests in the `test` directory, on the same level with `src`, it's conventional for us

Reply:
Ok I'll do that. Sorry, this is my way of placing tests collocated with their files so when you remove the file you remove the test. Anyway I'll move to tests 👍